### ISSUE TITLE
remote config

### ISSUE THREAD

**Issue by markus2330** [sirblackheart/elektrasurvey#40]
In which remote configuration scenarios would you be interested?

        through a web interface
        through ssh

Maybe better:

        configuration via a web interface
        configuration via ssh

maybe push configuration to question? (to make not every answer longer?)

**Comment by omnidan** [comment]
How about this:

## Question
In which remote configuration scenarios would you be interested?

Configuration...

## Answers
- via a web interface
- via ssh
- of groups/clusters of devices with similar configuration (e.g. multiple instances of web servers)
- of non-clustered devices with independent configurations
- of a single computer (e.g. configuring your home server)
- I'm not interested in remote configuration
- other: [textfield]

---

PS: where should we make changes now? questions.md, directly in limesurvey (which survey?), or both?

**Comment by markus2330** [comment]
PS: see  #46

**Comment by sirblackheart** [comment]
Feedback zur Frage:
> "In which remote configuration scenarios would you be interested?" - Frage kommt irgendwie aus dem Nichts. Muss ich die Konfigurationen remote machen? Hab ich die Wahl zwischen remote und nicht remote? Gehts da um mich als Benutzer oder um mich als Entwickler?

**Comment by markus2330** [comment]
Yeah, we should put a short sentence what remote configuration is before the question.

**Comment by omnidan** [comment]
I changed the question:

```
Imagine you have one or multiple machines that you want to configure/setup without having physical access. In which remote configuration scenarios would you be interested?

Remote configuration...
```

**Comment by markus2330** [comment]
I think the "Remote configuration..." is not needed and also does not fit in the style with the other questions.

**Comment by markus2330** [comment]
And there should be a newline after "access.".

**Comment by omnidan** [comment]
updated

**Comment by Namoshek** [comment]
Changes reviewed, but some answers don't really fit the question (anymore) without the "Remote configuration..." part. Example:
> In which remote configuration scenarios would you be interested?
>  -> of groups/clusters of devices with similar configuration (e.g. multiple instances of web servers)
> -> of non-clustered devices with independent configurations 
> -> of a single computer (e.g. configuring your home server)

**Comment by markus2330** [comment]
of -> in should solve the problem. (I am interested in...)

**Comment by Namoshek** [comment]
No, it would actually be "I am interested in **configuring** groups/clusters...", that's why I think the answers are weird. But adding "configuring" to every answer would be really stupid too.

> I am interested in non-clustered devices with independent configurations

Seems to be the answer to another question for me.

**Comment by omnidan** [comment]
@Namoshek If you prefix all the answers with `Remote configuration...` they sound fine (except the last one, but that's fine IMO). Maybe we should add this after the question: `In which remote configuration scenarios would you be interested? Remote configuration...`

Also, I think we should change the last answer to `I already use:`

**Comment by markus2330** [comment]
We still should still aim for:

- Not having "Remote configuration..." which is inconsistent
- Not having full sentences as answer (again inconsistent)

of -> in, and "not interested" is still the best option. But I have no doubts it can be formulated much better by rewriting the answers.

**Comment by omnidan** [comment]
I updated the answers again.

**Comment by Namoshek** [comment]
Looks ok to me now!
